src/utils/account/account.service.ts

```diff
@@ -81,6 +87,8 @@ export class AccountService {
     if (!upnDomain)
       throw new Error('Unable to identify the upn domain to be used');
 
+    const normalizer = require('normalizer');
```

Comment:
Require is not recommended in TS code. You either need to suppress the warning (not recommended) or import the type. I couldn't find @types/normalize so we either need to write our own type definition or suppress the eslint warning here (again, not recommended, but it will be faster).

Reply:
for now we just supress the eslint warning. Need to focus elsewhere.